packages/browser/src/core/user/index.ts

```diff
@@ -63,7 +63,7 @@ const ONE_YEAR = 365
 
 export class Cookie extends Store {
   static available(): boolean {
-    let cookieEnabled = window.navigator.cookieEnabled
+    let cookieEnabled = window && window.navigator.cookieEnabled
```

Comment:
Would be better to use the getGlobal() helper

 https://github.com/segmentio/analytics-next/blob/master/packages/core/src/utils/get-global.ts

Reply:
Sorry for taking so long, we ended up migrating away from the legacy analytics-node so this was no longer a breaking problem for us, but I'm including the recommended fix here since others might still have it.